eng/platform-matrix.yml

```diff
@@ -55,21 +55,21 @@ jobs:
         containerName: ubuntu_1604_arm64_cross_build_image
         helixQueues:
           ${{ if and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest', 'IndividualCI', 'BatchedCI')) }}:
-            asString: 'Ubuntu.1804.Arm64.Open'
+            asString: '(Ubuntu.18.04.Arm64.Open)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-0a0ebdd-20190312220351'
             asArray:
-            - Ubuntu.1804.Arm64.Open
+            - (Ubuntu.18.04.Arm64.Open)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-0a0ebdd-20190312220351
```

Comment:
Can you please move `mcr.microsoft.com/dotnet-buildtools/prereqs` into a variable on top of this file (e.g. `containerRegistry` ?

Also can you please move `ubuntu-18.04-helix-arm64v8-0a0ebdd-20190312220351` into something like `ubuntu1804Arm64DockerImage`? (and do the same for a Debian image)

And use these variables when defining queues here
For example, 
`(Ubuntu.1804.Arm64.On.Docker.Open)Ubuntu.1604.Arm64.Docker.Open@$(containerRegistry):$(ubuntu1804Arm64DockerImage)`
I like this string in parenthesis slightly more, but if you want you can keep `Ubuntu.18.04.Arm64.Open`